fleetctl/unload.go

```diff
@@ -42,6 +42,11 @@ func runUnloadUnit(args []string) (exit int) {
 		return 1
 	}
 
+	if len(units) == 0 {
+		stderr("No correct unit is found.")
```

Comment:
_correct_ still feels like the wrong word (i.e.: who is the judge?). I'd phrase this as: "The specified unit or units were not found."